crates/bdk/src/wallet/signer.rs

```diff
@@ -523,12 +574,10 @@ fn sign_psbt_schnorr(
     pubkey: XOnlyPublicKey,
     leaf_hash: Option<taproot::TapLeafHash>,
     psbt_input: &mut psbt::Input,
-    hash: taproot::TapSighashHash,
-    hash_ty: SchnorrSighashType,
+    hash: sighash::TapSighash,
```

Comment:
Stylistic choice but I'd do 
```suggestion
    hash: TapSighash,
```